sdk/storage/storage-blob/test/blobbatch.spec.ts

```diff
@@ -59,17 +61,16 @@ describe("BlobBatch", () => {
   });
 
   afterEach(async function (this: Context) {
-    if (!this.currentTest?.isPending()) {
+    if (containerClient) {
       await containerClient.delete();
-      await recorder.stop();
     }
+    await recorder.stop();
   });
 
-  it("submitBatch should work for batch delete", async () => {
-    recorder.skip(
-      undefined,
-      "UUID is randomly generated within the SDK and used in the HTTP request and cannot be preserved."
-    );
+  it("submitBatch should work for batch delete", async function () {
+    if (!isLiveMode()) {
```

Comment:
I'll log an issue for a future PR.
We can use sanitizers/transforms supported by the test-proxy to be able to handle the generated uuids at the SDK.

Reply:
https://github.com/Azure/azure-sdk-for-js/issues/24620